app/controllers/support/feature_flags_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Support
+  class FeatureFlagsController < ApplicationController
+    def update
+      FeatureFlag.send(action, feature_name)
```

Comment:
`action` and  `feature_name` comes directly from user params. 🤔 

I know that is an admin interface, but this is a security risk because it allows users to invoke any method on the FeatureFlag class, potentially leading to arbitrary code execution.

Can we use a before_action :verify_allowed_actions that verifies all possible actions before calling send?

Reply:
Great idea! :) I'll update.

Reply:
Decided to split out the `activate` and `deactivate` actions. The user will see be met with a 404 error in the case of sending an invalid action.